src/gdpr-banner/gdpr-banner.scss

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+@import '../dialog/index';
+
+@mixin iui-gdpr-banner {
+  @include iui-dialog;
+  border-radius: 0;
+  box-shadow: none;
+  border-top: 1px solid var(--iui-color-background-border);
+  border-right: 1px solid var(--iui-color-background-border);
+  border-left: 1px solid var(--iui-color-background-border);
+  position: fixed;
+  justify-content: center;
+  bottom: 0;
+  max-width: 75ch;
+}
+
+@mixin iui-gdpr-banner-title {
+  @include iui-dialog-title;
+}
+
+@mixin iui-gdpr-banner-title-bar {
+  @include iui-dialog-title-bar;
+}
+
+@mixin iui-gdpr-banner-content {
+  @include iui-dialog-content;
+}
+
+@mixin iui-gdpr-banner-button-bar {
+  @include iui-dialog-button-bar;
+}
```

Comment:
For all of the mixins that only contain `@include ...` I think you should be able to remove those mixins and instead just place, for example `iui-dialog-button-bar` where you now have `iui-gdpr-banner-button-bar`. Then you can also remove the declarations in the gdpr-banner/classes.scss file. Unless you were asked to do it this way?

Reply:
Yes, the styling for the banner is closely related to the dialog just to keep everything standard and consistent with the rest of the components. I understand what you are saying about replacing the mixins, could you give me an example?

Reply:
That is a good point Leah!  I may have told you to do it this way, but Leah's suggestion makes more sense.

Line 6, `@mixin iui-gdpr-banner`, uses `@include iui-dialog;` and has additional CSS (lines 7-16), so that one makes sense to leave as is.  The rest of the mixins only use a mixin & have no extra CSS, so we don't need to have a mixin reference an include with no additional CSS.

To achieve this:
- You can remove all the mixins that have no additional CSS.
- Within `classes.scss` update the classes to include the dialog mixins directly except for the one mixin that has custom CSS.  Screenshot shows how `classes.scss` should look like:
![Screen Shot 2022-06-01 at 1 28 00 PM](https://user-images.githubusercontent.com/849817/171466148-4f6258b8-92b2-483e-9fef-290bec971120.png)

Reply:
Got it fixed, thanks guys!